<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1024</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1024</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>El Gui Comrado v 0.4</string>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::TabShape::Rounded</enum>
  </property>
  <property name="dockNestingEnabled">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="System_frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>251</width>
      <height>571</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Raised</enum>
    </property>
    <widget class="QPushButton" name="Editor_button">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>460</y>
       <width>231</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Editor</string>
     </property>
    </widget>
    <widget class="QLCDNumber" name="lcdNumber">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>60</y>
       <width>111</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="Bar_frame">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>500</y>
      <width>730</width>
      <height>81</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Raised</enum>
    </property>
    <widget class="QGroupBox" name="Bar_groupBox">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>730</width>
       <height>81</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 4px double #444;        /* Толщина, стиль и цвет рамки */
    border-radius: 6px;             /* Закругление углов */
    background-color: #3c3c3c;      /* Цвет фона */
    padding: 5px;                  /* Внутренний отступ */
    margin-top: 0 px;               /* Отступ над заголовком */</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QStackedWidget" name="Button_stackedWidget">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>10</y>
      <width>750</width>
      <height>450</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>750</width>
      <height>450</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>750</width>
      <height>450</height>
     </size>
    </property>
    <property name="baseSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="focusPolicy">
     <enum>Qt::FocusPolicy::ClickFocus</enum>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::ContextMenuPolicy::CustomContextMenu</enum>
    </property>
    <property name="locale">
     <locale language="Russian" country="Russia"/>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::NoFrame</enum>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page_1">
     <widget class="QFrame" name="Button_frame_01">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>730</width>
        <height>440</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>730</width>
        <height>440</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">gridline-color: rgb(255, 170, 255);
border-color: rgb(255, 0, 255);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Shape::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Raised</enum>
      </property>
      <widget class="QToolButton" name="ToolButton_01">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>20</y>
         <width>128</width>
         <height>128</height>
        </rect>
       </property>
       <property name="focusPolicy">
        <enum>Qt::FocusPolicy::StrongFocus</enum>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="iconSize">
        <size>
         <width>96</width>
         <height>96</height>
        </size>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
       <property name="autoRepeatDelay">
        <number>600</number>
       </property>
       <property name="autoRepeatInterval">
        <number>300</number>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonStyle::ToolButtonTextUnderIcon</enum>
       </property>
      </widget>
      <widget class="QToolButton" name="ToolButton_02">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>20</y>
         <width>128</width>
         <height>128</height>
        </rect>
       </property>
       <property name="focusPolicy">
        <enum>Qt::FocusPolicy::StrongFocus</enum>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="iconSize">
        <size>
         <width>96</width>
         <height>96</height>
        </size>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
       <property name="autoRepeatDelay">
        <number>600</number>
       </property>
       <property name="autoRepeatInterval">
        <number>300</number>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonStyle::ToolButtonTextUnderIcon</enum>
       </property>
      </widget>
      <widget class="QToolButton" name="ToolButton_03">
       <property name="geometry">
        <rect>
         <x>390</x>
         <y>20</y>
         <width>128</width>
         <height>128</height>
        </rect>
       </property>
       <property name="focusPolicy">
        <enum>Qt::FocusPolicy::StrongFocus</enum>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="iconSize">
        <size>
         <width>96</width>
         <height>96</height>
        </size>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
       <property name="autoRepeatDelay">
        <number>600</number>
       </property>
       <property name="autoRepeatInterval">
        <number>300</number>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonStyle::ToolButtonTextUnderIcon</enum>
       </property>
      </widget>
      <widget class="QToolButton" name="ToolButton_04">
       <property name="geometry">
        <rect>
         <x>560</x>
         <y>20</y>
         <width>128</width>
         <height>128</height>
        </rect>
       </property>
       <property name="focusPolicy">
        <enum>Qt::FocusPolicy::StrongFocus</enum>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="iconSize">
        <size>
         <width>96</width>
         <height>96</height>
        </size>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
       <property name="autoRepeatDelay">
        <number>600</number>
       </property>
       <property name="autoRepeatInterval">
        <number>300</number>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonStyle::ToolButtonTextUnderIcon</enum>
       </property>
      </widget>
      <widget class="QToolButton" name="ToolButton_05">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>160</y>
         <width>128</width>
         <height>128</height>
        </rect>
       </property>
       <property name="focusPolicy">
        <enum>Qt::FocusPolicy::StrongFocus</enum>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="iconSize">
        <size>
         <width>96</width>
         <height>96</height>
        </size>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
       <property name="autoRepeatDelay">
        <number>600</number>
       </property>
       <property name="autoRepeatInterval">
        <number>300</number>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonStyle::ToolButtonTextUnderIcon</enum>
       </property>
      </widget>
      <widget class="QToolButton" name="ToolButton_06">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>160</y>
         <width>128</width>
         <height>128</height>
        </rect>
       </property>
       <property name="focusPolicy">
        <enum>Qt::FocusPolicy::StrongFocus</enum>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="iconSize">
        <size>
         <width>96</width>
         <height>96</height>
        </size>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
       <property name="autoRepeatDelay">
        <number>600</number>
       </property>
       <property name="autoRepeatInterval">
        <number>300</number>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonStyle::ToolButtonTextUnderIcon</enum>
       </property>
      </widget>
      <widget class="QToolButton" name="ToolButton_07">
       <property name="geometry">
        <rect>
         <x>390</x>
         <y>160</y>
         <width>128</width>
         <height>128</height>
        </rect>
       </property>
       <property name="focusPolicy">
        <enum>Qt::FocusPolicy::StrongFocus</enum>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="iconSize">
        <size>
         <width>96</width>
         <height>96</height>
        </size>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
       <property name="autoRepeatDelay">
        <number>600</number>
       </property>
       <property name="autoRepeatInterval">
        <number>300</number>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonStyle::ToolButtonTextUnderIcon</enum>
       </property>
      </widget>
      <widget class="QToolButton" name="ToolButton_08">
       <property name="geometry">
        <rect>
         <x>560</x>
         <y>160</y>
         <width>128</width>
         <height>128</height>
        </rect>
       </property>
       <property name="focusPolicy">
        <enum>Qt::FocusPolicy::StrongFocus</enum>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="iconSize">
        <size>
         <width>96</width>
         <height>96</height>
        </size>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
       <property name="autoRepeatDelay">
        <number>600</number>
       </property>
       <property name="autoRepeatInterval">
        <number>300</number>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonStyle::ToolButtonTextUnderIcon</enum>
       </property>
      </widget>
      <widget class="QToolButton" name="ToolButton_09">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>300</y>
         <width>128</width>
         <height>128</height>
        </rect>
       </property>
       <property name="focusPolicy">
        <enum>Qt::FocusPolicy::StrongFocus</enum>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="iconSize">
        <size>
         <width>96</width>
         <height>96</height>
        </size>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
       <property name="autoRepeatDelay">
        <number>600</number>
       </property>
       <property name="autoRepeatInterval">
        <number>300</number>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonStyle::ToolButtonTextUnderIcon</enum>
       </property>
      </widget>
      <widget class="QToolButton" name="ToolButton_10">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>300</y>
         <width>128</width>
         <height>128</height>
        </rect>
       </property>
       <property name="focusPolicy">
        <enum>Qt::FocusPolicy::StrongFocus</enum>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="iconSize">
        <size>
         <width>96</width>
         <height>96</height>
        </size>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
       <property name="autoRepeatDelay">
        <number>600</number>
       </property>
       <property name="autoRepeatInterval">
        <number>300</number>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonStyle::ToolButtonTextUnderIcon</enum>
       </property>
      </widget>
      <widget class="QToolButton" name="ToolButton_11">
       <property name="geometry">
        <rect>
         <x>390</x>
         <y>300</y>
         <width>128</width>
         <height>128</height>
        </rect>
       </property>
       <property name="focusPolicy">
        <enum>Qt::FocusPolicy::StrongFocus</enum>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="iconSize">
        <size>
         <width>96</width>
         <height>96</height>
        </size>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
       <property name="autoRepeatDelay">
        <number>600</number>
       </property>
       <property name="autoRepeatInterval">
        <number>300</number>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonStyle::ToolButtonTextUnderIcon</enum>
       </property>
      </widget>
      <widget class="QToolButton" name="ToolButton_12">
       <property name="geometry">
        <rect>
         <x>560</x>
         <y>300</y>
         <width>128</width>
         <height>128</height>
        </rect>
       </property>
       <property name="focusPolicy">
        <enum>Qt::FocusPolicy::StrongFocus</enum>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="iconSize">
        <size>
         <width>96</width>
         <height>96</height>
        </size>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
       <property name="autoRepeatDelay">
        <number>600</number>
       </property>
       <property name="autoRepeatInterval">
        <number>300</number>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonStyle::ToolButtonTextUnderIcon</enum>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QFrame" name="Page_control_frame">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>460</y>
      <width>255</width>
      <height>35</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Raised</enum>
    </property>
    <widget class="QLabel" name="Current_page_label">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>5</y>
       <width>60</width>
       <height>25</height>
      </rect>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::ContextMenuPolicy::ActionsContextMenu</enum>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Plain</enum>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="NextButton_main">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>5</y>
       <width>75</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>TempNext</string>
     </property>
    </widget>
    <widget class="QPushButton" name="BackButton_main">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>5</y>
       <width>75</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>TempBack</string>
     </property>
    </widget>
   </widget>
  </widget>
  <action name="action">
   <property name="iconText">
    <string/>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
